docs/PRPs/containers/mosquitto.md

```
@@ -0,0 +1,158 @@
+## Base information for Eclipse Mosquitto application
+
+Application name: Eclipse Mosquitto
+Homepage: https://mosquitto.org/
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)